applications/ShapeOptimizationApplication/tests/algorithm_penalized_projection_test/parameters.json

```diff
@@ -63,10 +63,19 @@
         },
         "output" : {
             "design_output_mode" : "WriteDesignSurface",
-            "nodal_results"      : [ "SHAPE_CHANGE" ],
+            "nodal_results"      : [
+                "SHAPE_CHANGE",
+                "DF1DX",
+                "DF1DX_MAPPED",
+                "DC1DX",
+                "DC1DX_MAPPED",
+                "SEARCH_DIRECTION",
+                "SHAPE_UPDATE"
+            ],
             "output_format" : {
```

Comment:
It would make sense to not require the output_format settings if `write_design_outout=false`.

Reply:
The idea here is that it should be easy to switch on the output for each test whenever needed.
By removing the related part one has to copy the settings (that typically no one knows by heart from somewhere) thats a lot of extra effort.